chukwuemekachm/utility_manager_web
# Prof: Adding forgot password logic to UI

#### What does this PR achieve?
- adds an implementation of Forgot Password
- adds a link expired page for ConfirmResetPassword
- extends the handleAuthOperation function in AuthenticationContainer
- adds reducers, and saga for the new component

#### Any background context?
- This is the first step of the reset password process
- Since the Login implementation has not yet been added, this feature shows the `Forgot password` by default. This would change when the login is merged

#### Manual Testing Steps
- When you start the app you should see the form
- When you input a valid email then reset password instructions should be sent to your email
- When click on the email link then you should be taken to the confirm password page

#### Screenshots:
<img width="1440" alt="Screenshot 2019-12-27 at 7 21 37 PM" src="https://user-images.githubusercontent.com/38565349/71528146-d800c000-28de-11ea-8cb5-127ae75adcc6.png">
<img width="1440" alt="Screenshot 2019-12-27 at 7 21 31 PM" src="https://user-images.githubusercontent.com/38565349/71528148-d800c000-28de-11ea-9aae-8664365deea6.png">

#### Relevant Trello Tickets
- [Forgot Password](https://trello.com/c/E3CGVrer/34-user-should-reset-password)
